src/main/java/io/fabric8/maven/docker/StopMojo.java

```diff
@@ -71,6 +72,41 @@
     @Parameter(property = "docker.stopNamePattern")
     private String stopNamePattern;
 
+    /**
+     * If true, the containers are not stopped right away, but when the build is finished (success or failed).
+     */
+    @Parameter(property = "docker.executeStopOnVMShutdown", defaultValue = "false")
+    private boolean executeStopOnVMShutdown;
```

Comment:
Would appreciate if you could add documentation regarding this field in `docker:stop` configuration[0]. Source for this doc section is located here[1]

[0] http://dmp.fabric8.io/#docker:stop
[1] https://github.com/fabric8io/docker-maven-plugin/blob/master/src/main/asciidoc/inc/_docker-stop.adoc